com.avaloq.tools.ddk.xtext.scope.generator/build.properties

```diff
@@ -1,4 +1,5 @@
-source.. = src/
+source.. = src/,\
+		   xtend-gen/

```

Comment:
Mix of tabs and spaces. It's helpful to enable "show whitespace characters" in Eclipse preferences for "text editor".